rfc-174-consolidate-frontend-rendering-apps.md

```diff
@@ -0,0 +1,308 @@
+---
+status: proposed
+implementation: proposed
+status_last_reviewed:
+---
+
+# Consolidate Frontend Apps
+
+## Summary
+We have too many frontend rendering applications. This increases complexity, reduces understanding and slows development. We propose consolidating our apps into a single app. This will improve standardisation, reduce maintenance costs and improve the developer experience.
+
+## Problem
+
+### Applications overview
+GOV.UK is served by multiple rendering applications (apps). Each app serves different groups of pages within the website.
+
+There are currently 8 rendering apps for GOV.UK that are supported and maintained by GOV.UK teams:
```

Comment:
Although if you accept @kevindew 's suggestion that we focus this RFC on the top 4 most obviously mergeable apps, it's fine to ignore these edge cases.

Reply:
The APIs were ignored as they don't render what I'd think of user facing content, i.e. web "pages". But yes, given that those routes all follow a pattern, it would be easy to add a redirect for them in frontend.

Licensify is also out of scope as it's a Scala app that we (GOV.UK teams) don't maintain. I'd be happy for it to move to it's own domain though.